examples/flutter_gallery/.idea/runConfigurations/main_dart.xml

```diff
@@ -1,5 +1,5 @@
 <component name="ProjectRunConfigurationManager">
-  <configuration default="false" name="flutter_gallery" type="FlutterRunConfigurationType" factoryName="Flutter">
+  <configuration default="false" name="main.dart" type="FlutterRunConfigurationType" factoryName="Flutter">
```

Comment:
I think the template is good. For the flutter repo, we have several modules in one project, and it will look odd if all the launch configurations are named `main.dart`.